flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql/fun/SqlRowOperator.java

```diff
@@ -66,12 +70,17 @@
  *       </ul>
  * </ul>
  *
- * Once Flink applies same logic for both table api and sql, this class should be removed.
+ * <p>Once Flink applies same logic for both table api and sql, this first changes should be
+ * removed.
+ *
+ * <p>2. It uses PEEK_FIELDS_NO_EXPAND with a nested struct type (Flink [[RowType]]).
+ *
+ * <p>See more at {@code LogicalRelDataTypeConverter} and {@code FlinkTypeFactory}.
```

Comment:
```suggestion
 * <p>See more at {@link LogicalRelDataTypeConverter} and {@link FlinkTypeFactory}.
```
i guess it's better to use `@link` here